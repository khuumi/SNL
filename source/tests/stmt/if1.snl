if local x = 5
(do func1 to local x, 7.9)