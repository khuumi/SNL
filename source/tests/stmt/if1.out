If (Binop (Id Local x) Equal (Int 5)) (Block [(Expr (Call func1 [(Id Local x), (Float 7.9)]))]) (Block [])